include/kernel/ebpf_registry_helper.h

```diff
@@ -6,12 +6,14 @@
 #include <ntddk.h>
 
 #define __return_type NTSTATUS
-#define IS_SUCCESS(x) (NT_SUCCESS(x))
 #define _SUCCESS STATUS_SUCCESS
+#define IS_SUCCESS(x) (NT_SUCCESS(x))
 
 #define REG_CREATE_FLAGS 0
 #define GUID_STRING_LENGTH 38 // not including the null terminator.
 
+typedef _Return_type_success_(NT_SUCCESS(return )) uint32_t ebpf_registry_result_t;
```

Comment:
bug
```suggestion
typedef _Return_type_success_(NT_SUCCESS(return == 0)) uint32_t ebpf_registry_result_t;
```

Reply:
Not sure I follow. NT_SUCCESS is defined as a comparison of >= 0:
```
#define NT_SUCCESS(Status)  (((NTSTATUS)(Status)) >= 0)
```